forc-plugins/forc-client/src/lib.rs

```diff
@@ -14,19 +14,27 @@ pub struct NodeTarget {
     /// If unspecified, checks the manifest's `network` table, then falls back
     /// to `http://127.0.0.1:4000`
     ///
-    /// You can also use `--target` or `--testnet` to specify the Fuel node.
+    /// You can also use `--target`, `--testnet`, or `--mainnet` to specify the Fuel node.
     #[clap(long, env = "FUEL_NODE_URL")]
     pub node_url: Option<String>,
+
     /// Use preset configurations for deploying to a specific target.
     ///
-    /// You can also use `--node-url` or `--testnet` to specify the Fuel node.
+    /// You can also use `--node-url`, `--testnet`, or `--mainnet` to specify the Fuel node.
     ///
-    /// Possible values are: [beta-1, beta-2, beta-3, beta-4, local]
+    /// Possible values are: [beta-1, beta-2, beta-3, beta-4, devnet, local, testnet, mainnet]
```

Comment:
Let's remove the beta's and devnet 

```suggestion
    /// Possible values are: [local, testnet, mainnet]
```

Reply:
added in https://github.com/FuelLabs/sway/pull/6634